packages/web/src/components/edit/fields/stream-availability/usdc-purchase-gated/UsdcPurchaseFields.tsx

```diff
@@ -273,6 +305,31 @@ const PriceField = (props: PriceFieldProps) => {
       onChange={handlePriceChange}
       onBlur={handlePriceBlur}
       disabled={disabled}
-    />
+    >
+      {shouldShowRightsDeclaration && (
```

Comment:
It feels a _little_ strange to mix publishing rights into a field that seems designed for general use of inputting a price value on a form. I'm wondering how hard it would be to move this outside of PriceField and compose in the additional UI components.

If it's prohibitively difficult, then probably not worth it.